packages/graphql/src/typeDefs/Growth.js

```diff
@@ -221,5 +224,9 @@ module.exports = `
     log(event: JSON!): Boolean
     # Remind a user that his invitation is still pending
     inviteRemind(invitationId: Int!): Boolean
+    # Confirm executing social share
+    confirmSocialShare(contentId: String!, actionType: GrowthActionType!): Boolean
```

Comment:
super nit on the naming: this is not so much a "confirm" but more a "log" ?

Reply:
agreed that is more appropriate